app/assets/stylesheets/local/homepage.scss

```diff
@@ -171,4 +171,64 @@
             }
         }
     }
+
+    $featured-topics-box-width: 21.875rem;
+    $featured-topics-box-height: 21.875rem;
+    $featured-topics-box-margin: 3.6875rem;
```

Comment:
So I know I said to do everything in `rem`... but these are actually sized based on _our thumb image sizes_. 

I wonder if they should actually still be in px?  At least the height/width, maybe not the margin.  Especially cause width is used as a media query -- I havne't totally wrapped my head around what media queries in terms of `rem` do, if they actually work (and on all browsers). I think they may not work on some. 

I'm actually not totally sure, we could just go ahead and merge... but how about we change these back to `px`? 

And then feel free to go ahead and merge this PR yourself please, rest looks good!

Reply:
Changed back to px! no problem.